ajax/libs/jquery-appear/0.1/jquery.appear.js

```diff
@@ -0,0 +1,151 @@
+/*
```

Comment:
This file is different from upstream.
This file's md5sum is `cbbd68c06c9cdb2481b7050f6e64375e`.
Upstream's is `5a457d262e3c32d25c003ca412ee7fe6`
Please re-add the file.
Thanks.